contracts/infrastructure/helper/MultiCallHelper.sol

```diff
@@ -0,0 +1,148 @@
+// Copyright (C) 2018  Argent Labs Ltd. <https://argent.xyz>
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.6.12;
+pragma experimental ABIEncoderV2;
+
+import "../dapp/IFilter.sol";
+import "../storage/ITransferStorage.sol";
+
+interface IDappRegistry {
+    function enabledRegistryIds(address _wallet) external view returns (bytes32);
+    function authorisations(uint8 _registryId, address _dapp) external view returns (bytes32);
+}
+
+/**
+ * @title MultiCallHelper
+ * @notice Helper contract that can be used to check in 1 call if and why a sequence of transactions is authorised to be executed by a wallet.
+ * @author Julien Niset - <julien@argent.xyz>
+ */
+contract MultiCallHelper {
+
+    uint256 private constant MAX_INT = 2**256 - 1;
```

Comment:
MAX_INT -> MAX_UINT

Reply:
Same renaming has to happen in `test/multiCallHelper.js` as well.